### PR TITLE
Publish IBIS explainer

### PR DESCRIPTION
**Pull request by jcespejo**

### CODE REVIEW

File: proposals/ibis/README.md [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+### How is the maximum bid price provided to the on-device auction?
+
+After the DSP determines the maximum bid price for their creatives for the
+current contextual bid request, they can then include the bid price constraint
+information as part of the bid response sent back to the SSP, and subsequently
+forwarded to the on-device auction.
+
+The bid price constraint could look like the following:
+
+```
```
Comment (sbelov) [inline proposals/ibis/README.md:140]: nit: you can use jsonc syntax highlighting with ```jsonc, e.g.:

```jsonc
  "seatbid": [{
    ...
  }]
  // Signals…
  "ext": {
    "interest_group_bidding": {//…
    }
  }
```

File: proposals/ibis/README.md [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+    increase the probability of a preferred ad to be shown instead. These types
+    of threats can be mitigated by other solutions mentioned below, but will not
+    be the focus of IBIS.
+
+### Are contextual ad requests affected?
+
+In the TURTLEDOVE proposal, contextual ad requests bids will continue to be
+determined server-side. Contextual bid price can be cryptographically signed on
+the seller’s server and can be sent back as part of the tracker URL when
+notifying of the ad impression, similar to today’s state. Therefore there would
```
Comment (sbelov) [inline proposals/ibis/README.md:70]: maybe just "impression tracking URL"

File: proposals/ibis/README.md [hunk 3]
```diff
@@ -0,0 +1,189 @@
...
+
+*   Executing the bidding functions and the on-device auction in an isolated
+    sandbox environment would prevent code from outside the sandbox (such as
+    code from the publisher page, other participants in on-device auction,
+    malicious browser extensions) to interfere with the logic or the results of
+    the bidding function and the on-device auction. Isolated environments will
+    also prevent faulty code from one bidding function to affect another.
+*   Browsers could compute a hash of the bidding function code that can be made
+    available at impression reporting time to be used by the ad tech companies
+    to verify that the one of expected bidding functions computed the bid value.
```
Comment (sbelov) [inline proposals/ibis/README.md:186]: should this be "one of the expected…"?

**Review by sbelov — APPROVED** [review]

File: proposals/ibis/README.md (hunk 1 above)
Comment (jcespejo) [reply proposals/ibis/README.md]: Done

File: proposals/ibis/README.md (hunk 2 above)
Comment (jcespejo) [reply proposals/ibis/README.md]: Done

File: proposals/ibis/README.md (hunk 3 above)
Comment (jcespejo) [reply proposals/ibis/README.md]: Done

**Comment by jcespejo** [comment]
@googlebot I fixed it.

**Pull request closed without merging**